src/BasicBehaveEngine/nodes/math/matrix/Inverse.ts

```diff
@@ -0,0 +1,39 @@
+import {BehaveEngineNode, IBehaviourNodeProps} from "../../../BehaveEngineNode";
+import {MatrixHelper} from "../matrix/MatrixHelper";
+
+export class Inverse extends BehaveEngineNode {
```

Comment:
Invert matrix would be great! I skipped that one bc I didn't want to do the math